app/models/impression.rb

```diff
@@ -34,11 +34,12 @@ class Impression < ApplicationRecord
   # includes ..................................................................
 
   # relationships .............................................................
-  belongs_to :advertiser, class_name: "User", foreign_key: "advertiser_id"
-  belongs_to :publisher, class_name: "User", foreign_key: "publisher_id"
-  belongs_to :campaign
+  belongs_to :organization, optional: true
+  belongs_to :advertiser, class_name: "User", foreign_key: "advertiser_id", optional: true
+  belongs_to :publisher, class_name: "User", foreign_key: "publisher_id", optional: true
+  belongs_to :campaign, optional: true
   belongs_to :creative, optional: true
-  belongs_to :property
```

Comment:
Why would this be optional?

Reply:
I made these all optional to support more flexibility with the seeds and to prevent impressions from blocking other operations for the data they reference.